### DISCUSSION TITLE
Open Thread - Most Important Questions

### DISCUSSION THREAD

**Discussion by jwildfire** [SafetyGraphics/safetyGraphics#479] [General]
What do you think are the most important and challenging questions in safety analytics that our group should address in the coming year(s)?

**Comment by jwildfire** [comment]
One topic of interested is figuring out how to submit interactive graphics to the FDA.

**Reply by anzianorj** [reply]
This is definitely worthwhile exploring and I think that for FDA there may be openness to exploratory tools to go with the datasets.  It would also be great to also have the ability if one sees something in an interactive plot to be able to translate that to static code for inclusion as a figure in a dossier given that regulators outside the US may not have the capacity to do their own analyses and likely rely on sponsor to provide the views the old fashioned way as static figs

**Reply by samussiah** [reply]
It'd be pretty slick to snapshot the state of an interactive graphic that aligns with a static display, e.g. the filters or subgroups, the grouping, the outcome of interest, etc., basically connect the interactive graphic with the static display via metadata.

**Reply by jwildfire** [reply]
@samussiah It's very much a work in progress, but that's what we're going for in [safetyCharts](https://github.com/SafetyGraphics/safetyGraphics/). This [draft vignette](https://github.com/SafetyGraphics/safetyGraphics/wiki/Chart-Configuration-Vignette-(v2)) for v2 goes through some of the details. Any questions/feedback related to the vignette would be great!

**Reply by MikeColopy** [reply]
This is my first attempt after Jeremy's demo of GitHub.   I have a QLIK safety dashboard that handles all lab data and I will be helping to develop its twin using RShiny.  How do I avoid R not choking on the huge volume of lab data?  Is the trick to focus on specific toxicities such as liver or kidney or CV?

**Reply by zskrivan** [reply]
We already know how to submit a stand alone interactive HTML to the FDA by including this directly in the submission package (in a clever way so the experience is seamless).  We have included interactive data viz in two separate submissions as a pilot for this approach, working with PHUSE.  The PHUSE group is currently seeking an FDA sponsor to pilot a submission using containerization.  

Do you want to replicate the efforts of the Phuse group?  It seems that it would be more productive if we focused on a gap like 'how to validate our interactive visualization used in submission".  That could complement PHUSE's effort, rather than duplicate it.

**Reply by jwildfire** [reply]
> How do I avoid R not choking on the huge volume of lab data? 

This is a very good question @AlbrechtDurer, and in my experience the answer is complex and varies for different use cases. Focusing on specific toxicities helps, but probably isn't enough in really big studies. In those cases, I think the most important thing is to design a good data pipeline that includes both a database backend (as opposed to loading all of your study data each time you initialize the app) and visualizations that summarize the data in a reasonable way (as opposed to just plotting every single data point in a scatter plot no matter what). Fortunately this is all doable in R, and improvements in this area are our radar for safetyGraphics after v2 goes live. 

Note that I've added your question and my answer above to the FAQ section in the data pipeline and security vignette. Hopefully others can weigh in with additional guidance for this area.

**Reply by jwildfire** [reply]
> It seems that it would be more productive if we focused on a gap like 'how to validate our interactive visualization used in submission".

Agreed, @zskrivan! We've discussed reaching out to the folks at R Validation hub to see if they're interested in collaborating on this topic once we're a little bit closer to release.

**Comment by anzianorj** [comment]
A non-technical topic but still important to ensure these tools get used, would be to discuss our communication strategy.

**Reply by jwildfire** [reply]
I believe there is a communications team from the full working group that we could collaborate with. Maybe @xni7 or @mli1 could say more?

**Reply by xni7** [reply]
Correct, @jwildfire , we do have an Outreach and Communication Team (OCT) for the overall Safety Working Group. I'm a member of the OCT. 

See [http://bit.ly/asa-safety](http://bit.ly/asa-safety) "Who We Are" tab. 

Agreed that this would be a good topic, jointly with the OCT team, to come up with the outreaching strategy. ![](https://s3.amazonaws.com/higherlogicdownload/AMSTAT/UploadedImages/hmclK1Q6RMupBUmmaPgB_asa-swg-membership-jul2019.png)

**Reply by mli1** [reply]
@anzianorj  Rich do you have any suggestions or thoughts on how we can do better communication? It is always good to hear fresh thoughts and perspectives:)

**Reply by anzianorj** [reply]
Apologies, being a newbie to the group I wasn't aware that there was a comms team.  Very glad to hear there is, as I think that is where we fell short in previous groups I've been involved with.  Ultimately there needs to be a critical mass of users to create a sustained 'pull' otherwise our 'push' will fade over time.  No specific suggestions but always good to have posters or presentations at the bigger meetings.

**Comment by zskrivan** [comment]
One general safety topic is how to visually analyze 'digital health' data.  We are getting more and more safety related data that comes for iOT (internet of things) devices.  We should lead the industry in how we visualize this data (which will often involve modeling and data reduction techniques).

**Reply by jwildfire** [reply]
Interesting (and BIG) topic, @zskrivan.  I've dabbled with wearable data on a few projects and found that data architecture (database design, etc) ends up being just as important as the visualization techniques, so we may want to keep an eye out for people with that skill set.

**Comment by mli1** [comment]
How to visualize the three-line (RBC/Hb, platelet, WBC/neutrophil) change with regards to drug interruption/dose reduction? In reality, when a drug is known to cause myelosuppression, it is hard to believe there would be no action taken at all. From an aggregated level, is there a better to visualize the efforts by the drug and the efforts of intervention?

**Comment by aslimem** [comment]
> What do you think are the most important and challenging questions in safety analytics that our group should address in the coming year(s)?

I believe it would be useful to have a base tool developed in R that is also customizable and expandable. Our safety teams have settled on using Tableau dashboard for much of their safety reviews. Although Tableau was not intended for safety reviews, the Tableau folks in our company can very quickly develop dashboard and patient profiles. Using R it takes us a lot longer and it takes some time to make it look slick. The drawback of Tableau is that it isn't linked to powerful statistical tools like R is. A base R app that can summarize safety information and patient profiles but also be expandable so the user can add additional variables would be cool. JM2C

**Comment by mli1** [comment]
Jerry @tbajn  and I just added a chart example (#496 ) where we wanted to address the question regarding visualization of difference between/among the giving categorical factor (subgroups) with regard to the time to a particular AE of interest (or AEs), plus proportion of those who had the AE(s). We understand the AE volcano plot can do a lot, especially with regards to proportion and the hazard ratio or RR , but the current version is not able to address "time to onset" issue/ subgroup analysis?  

@xni7 @DOBrien1 Xiao and Dennis I will nudge you here because I know you are working on AE volcano plot. @JimBuchanan @hendricksonb feel free to weigh in as well.

**Comment by DOBrien1** [comment]
Hi Mengchun
Not sure if this is the question is but I can comment that I definitely find the KM value plots valuable.  I agree with you.  We use then frequently  visually comparing for example TTO and number at risk for a given event of interest for tx groups.  Can be plotted by subgroups.  I regard these analysis as standard for focused analyses.
Dennis

**Reply by hendricksonb** [reply]
Agree with Dennis about the KM plots being the standard way we would look at time to onset.  The issue with subgroup analyses at filing is that you have to have enough of the events of interest in each of the subgroups you are comparing to be able make a meaningful conclusion.  Otherwise you have to wait and look at larger post-marketing database studies

**Comment by mli1** [comment]
A QT subgroup is having ongoing conversations. Key notes are posted at #511.
